test/common/helpers.test.js

```diff
@@ -342,3 +367,101 @@ test('convertToMarthaV3Response should return null for googleServiceAccount if b
 
     t.deepEqual(convertToMarthaV3Response(mockDrsResponse), expectedResponse);
 });
+
+function testFailureResponse(t, res, expectedStatus, expectedText) {
+    t.true(Object.prototype.isPrototypeOf.call(FailureResponse.prototype, res.send.firstCall.firstArg));
```

Comment:
FYI @breilly2 googling the eslint error clarified the JS syntax I was supposed to be using:
- https://eslint.org/docs/rules/no-prototype-builtins
- https://stackoverflow.com/questions/18343545/javascript-isprototypeof-vs-instanceof-usage#comment26927518_18343687